docs/content/stack/extend-images.md

```diff
@@ -71,7 +71,7 @@ USER docker
 # Install additional global npm dependencies
 RUN \
 	# Initialize nvm environment
```

Comment:
```suggestion
	# Initialize the user environment (this loads nvm)
```

Reply:
Ping @charginghawk. Please accept the suggested change, so we can merge this PR and get you a commit credit :)